src/registrar/templates/domain_add_user.html

```diff
@@ -37,6 +37,9 @@
     </nav>
     {% endif %}
   {% endblock breadcrumb %}
+
+  {% include "includes/form_errors.html" with form=form %}
+
```

Comment:
I read on the ticket that the error is supposed to appear on the field level and at the top of the page. Was there an update that wasn't noted? I don't get the error at the top

<img width="697" alt="Screenshot 2024-12-23 at 10 24 24 AM" src="https://github.com/user-attachments/assets/585a2d3c-0294-4bfb-afe8-fd9560c0b7aa" />

Reply:
This wasn't deployed to MS at the time you did the review, sorry. I have two PR's up but forgot that I only have one sandbox 🫠. Should deploy shortly.